### PR TITLE
[jquery] Add `includeMargin` parameter to `.outerHeight()` and `.outerWidth()` setters.

### PR DESCRIPTION
**Pull request by leonard-thieu**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
https://github.com/jquery/jquery/blob/354f6036f251a3ce9b24cd7b228b4c7a79001520/test/unit/dimensions.js#L477-L484.
- ~~Increase the version number in the header if appropriate.~~
- ~~If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.~~

### CODE REVIEW

**Review by MrHen — APPROVED** [review]

File: types/materialize-css/carousel.d.ts [hunk 1]
```diff
@@ -111,7 +111,9 @@ declare namespace M {
 interface JQuery {
     carousel(method: keyof Pick<M.Carousel, "destroy">): JQuery;
     carousel(method: keyof Pick<M.Carousel, "next">, n?: number): JQuery;
+    // tslint:disable-next-line:unified-signatures
```
Comment (sheetalkamat) [inline types/materialize-css/carousel.d.ts]: Why? Please fix the signature

File: types/materialize-css/collapsible.d.ts [hunk 2]
```diff
@@ -78,6 +78,7 @@ declare namespace M {
 interface JQuery {
     collapsible(method: keyof Pick<M.Collapsible, "destroy">): JQuery;
     collapsible(method: keyof Pick<M.Collapsible, "open">, n: number): JQuery;
+    // tslint:disable-next-line:unified-signatures
```
Comment (sheetalkamat) [inline types/materialize-css/collapsible.d.ts]: Instead of this please fix the signature

File: types/materialize-css/character-counter.d.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ declare namespace M {
 }
 
 interface JQuery {
+    // tslint:disable-next-line:unified-signatures
```
Comment (sheetalkamat) [inline types/materialize-css/character-counter.d.ts]: Same here

File: types/materialize-css/range.d.ts [hunk 4]
```diff
@@ -21,5 +21,6 @@ declare namespace M {
 
 interface JQuery {
     range(): JQuery;
+    // tslint:disable-next-line:unified-signatures
```
Comment (sheetalkamat) [inline types/materialize-css/range.d.ts]: Fix the signature please.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

**Review by sheetalkamat — APPROVED** [review]

**Pull request merged**